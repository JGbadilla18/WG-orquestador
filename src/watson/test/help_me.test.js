const testWatson = require('./testWatson');

watsonText = 'Ayudame con mi gestion.'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
