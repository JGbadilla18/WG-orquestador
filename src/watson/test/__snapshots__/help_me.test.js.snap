// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ayudame con mi gestion. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Â¡Hola! Soy ABI, el asistente virtual 24/7 de Banco Industrial y siempre estoy de tu lado. ðŸ¤– Me han creado para ayudarte con tus dudas. Por ahora no puedo entender emojis, stickers, imÃ¡genes y notas de voz.",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
